code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -1000,15 +1000,14 @@ Pass a positive integer as an argument to override a bot's default speed.
 
 /mob/living/simple_animal/bot/handle_hud_icons_health()
 	..()
-	if(bodytemp)
```

Comment:
@Fox-McCloud it was a reference to the body temperature hud icon, not the body temperature